cmd/medius/publish/publish.go

```diff
@@ -1,4 +1,4 @@
-package main
```

Comment:
Shouldn't be the file called `cmd/medius/publish/images.go` ?

Reply:
Good point. Restructured a few things where this got changed too.